### PR TITLE
Fix Sonarcloud integration

### PR DESCRIPTION
**Pull request by louisjdmartin**
## Proposed changes

Fix sonarqube analysis in CI

## Types of changes

What types of changes does your code introduce to this software?
> _Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

> _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/cnescatlab/sonar-cnes-report/blob/master/CONTRIBUTING.md) doc
- [x] I agree with the [CODE OF CONDUCT](https://github.com/cnescatlab/sonar-cnes-report/blob/master/CONTRIBUTING.md)
- [x] Lint and unit tests pass locally with my changes
- [x] SonarCloud and Travis CI tests pass with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Topin2001 — COMMENTED** [review]
Is it mandatory to keep java 11 in the matrix ? Since we are only using 17 (if I'm correct)

File: pom.xml
Comment (Topin2001) [inline pom.xml:1]: I think we should update the version from to 4.20 to 4.3.0 here also

File: .github/workflows/java-continuous-integration.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '11' ]
+        java: [ '11', '17' ]
```
Comment (Topin2001) [inline .github/workflows/java-continuous-integration.yml:18]: Is it mandatory to keep java 11, since I think we are no longer using it ?

File: pom.xml
Comment (louisjdmartin) [reply pom.xml:1]: Done !

File: .github/workflows/java-continuous-integration.yml (hunk 1 above)
Comment (louisjdmartin) [reply .github/workflows/java-continuous-integration.yml:18]: We are only using java 17 for sonar-scanner.
Matrix defines tests for the cnes-report itself, which can run on java 11 (even if its now deprecated by SonarQube)

**Review by Topin2001 — APPROVED** [review]

**Pull request merged**